documents/Libraries/stdlib/stdlib_defs.mtlx

```diff
@@ -56,9 +67,7 @@
     <parameter name="file" type="filename" uiname="Filename"/>
     <parameter name="layer" type="string" value="" uiname="Layer"/>
     <parameter name="default" type="float" value="0.0" uiname="Default Color"/>
-    <input name="texcoord" type="vector2" uiname="Texture Coordinates">
-      <geomprop name="texcoord" type="vector2"/>
-    </input>
+    <input name="texcoord" type="vector2" defaultgeomprop="UV0" uiname="Texture Coordinates"/>
```

Comment:
Revert back to use the old defaultgeomprop attribute, but now referencing a geompropdef element.

Reply:
What happens if the geompropdef type doesn't match the type specified for the input? e.g. if UV0 was a vector3?

Reply:
The type specifier on geompropdef was a request from ILM (from the Slack discussion), to be extra clear on what type of data it is. We don't use it in ShaderX as we use the data type from the input itself instead. 

And I think we should keep ignoring the type and treat it more like documentation info. Like you point out some geometric nodes can hold multiple types, so it needs to be optional, otherwise we would need multiple definitions. Might be worth discussing this more with ILM, any maybe remove the type specifier instead.

I think the best solution for now is to remove it in this change list and then we can add it in later if ILM really wants it. I'll make the change.